src/Orchard.Web/Modules/Orchard.MediaLibrary/Views/Folder/Create.cshtml

```diff
@@ -4,29 +4,31 @@
 }
 
 <div class="breadCrumbs">
-    <p>@Html.ActionLink(T("Media Library").ToString(), "Index", "Admin", new { area = "Orchard.MediaLibrary" }, new {}) &#62; 
+    <p>
+        @Html.ActionLink(T("Media Library").ToString(), "Index", "Admin", new { area = "Orchard.MediaLibrary" }, new { }) &#62;
     @if (Model.FolderPath != null) {
         var fullPath = "";
         foreach (var folder in Model.FolderPath.Split(new[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar })) {
             if (!String.IsNullOrEmpty(folder)) {
                 fullPath = Path.Combine(fullPath, folder);
-                @Html.ActionLink(folder, "Index", "Admin", new {folderPath = fullPath }, null)
-                    <text>&#62;</text>
-             }
-         }
+                @Html.ActionLink(folder, "Index", "Admin", new { folderPath = fullPath }, null)
+                <text>&#62;</text>
+            }
+        }
     }
-    @T("Add a Folder") </p>
-</div> 
+    @T("Add a Folder")
+</p>
+</div>
 
-@using (Html.BeginFormAntiForgeryPost()) { 
+@using (Html.BeginFormAntiForgeryPost()) {
     @Html.ValidationSummary()
     <fieldset>
         <label for="Name">@T("Folder Name")</label>
-        <input id="Name" class="text medium" name="Name" type="text" autofocus="autofocus"/>
+        <input id="Name" class="text medium" name="Name" type="text" autofocus="autofocus" pattern="@Model.InvalidCharactersPattern" title="@T("The name contains invalid characters.")" />
```

Comment:
Also, if you just want to encode the pattern, don't expose a property to do that, encode based on the context, here from the View directly, you can use JavascriptEncode.